### PR TITLE
swaylock-effects: 1.6-2 -> 1.6-3

### PR DESCRIPTION
**Pull request by ymatsiuk**
###### Motivation for this change
minor version bump + fmt

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by bbigras** [comment]
Result of `nixpkgs-review pr 119869` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>swaylock-effects</li>
  </ul>
</details>

File: pkgs/applications/window-managers/sway/lock-effects.nix [hunk 1]
```diff
@@ -1,18 +1,27 @@
...
+, libxkbcommon
+, cairo
+, gdk-pixbuf
+, pam
 }:
 
 stdenv.mkDerivation rec {
   pname = "swaylock-effects";
-  version = "v1.6-2";
+  version = "v1.6-3";
```
Comment (SuperSandro2000) [inline pkgs/applications/window-managers/sway/lock-effects.nix:18]:
```suggestion
  version = "1.6-3";
```

File: pkgs/applications/window-managers/sway/lock-effects.nix [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
 
   src = fetchFromGitHub {
     owner = "mortie";
     repo = "swaylock-effects";
     rev = version;
```
Comment (SuperSandro2000) [inline pkgs/applications/window-managers/sway/lock-effects.nix:23]:
```suggestion
    rev = "v${version}";
```

**Comment by Ma27** [comment]
@ymatsiuk thanks!

Decided to backport this (7c23e16b65694437fcfbc4101755934cc0f56213) and the previous bump (e3307d287131a5c560100363cd548858e9c5ecd6) to 20.09. Both seem to have relevant bugfixes and no breaking changes.

**Pull request merged**